README.md

```diff
@@ -139,7 +139,7 @@ These external dependencies are shipped with Layman for development, testing and
 However, if you want to run Layman in production, it is strongly recommended to install external dependencies separately. Recommended (i.e. tested) versions are:
 - PostgreSQL 13.3 & PostGIS 3.1
 - QGIS Server 3.40.4
-- GeoServer 2.21.4
+- GeoServer 2.26.2
```

Comment:
Search for `2.21` in the whole project and replace appropriate occurrences with `2.26`. E.g. in `.remarkrc`, `*.md`, ...